package.json

```diff
@@ -95,6 +95,7 @@
     "fast-safe-stringify": "^2.0.7",
     "flatstr": "^1.0.12",
     "get-caller-file": "^2.0.5",
+    "pino-abstract-transport": "^0.1.0",
```

Comment:
This commit adds this dependency but doesn't use it? I'd also prefer we bump it to 1.0.0 before shipping.

Reply:
I'm working on it, I committed it too soon.